jme3-core/src/main/java/com/jme3/anim/tween/action/Action.java

```diff
@@ -151,6 +151,11 @@ public AnimationMask getMask() {
     /**
      * Internal use-only.
      *
+     * <p> 
+     * Note: This method can be invoked from the user code if this Action is wrapped by a {@link BaseAction} and
+     * the {@link BaseAction#isMaskPropagationEnabled()} is false.
+     * </p>
+     *
```

Comment:
I believe this note is better and it's compatible with all the implementations of `Action`.

Reply:
Paul suggested changing “for internal use only” to “meant for internal use”, it is not good that you always try to ignore what others sugested!

Reply:
I am not ignoring what Paul has confirmed, this is just an assertive note for the only condition this could be used on the user code, it's literally what we were talking about the last review, do you see this note as confusing?

Reply:
It is a summary of what you have explained previously: 
> Someone might want to create a sequence of multiple actions and wants each action to have a different mask (for example wants the first one to run on the right hand and the second one to run on the head) then he must specify the mask on each action when creating them.

If you think it's confusing or an untrue assertion, I will remove it.